engine/test-utils/src/main/java/io/deephaven/engine/testutil/TstUtils.java

```diff
@@ -173,11 +175,12 @@ public static void addToTable(final Table table, final RowSet rowSet, final Colu
             }
         }
 
-        if (!usedNames.containsAll(table.getDefinition().getColumnNameSet())) {
-            final Set<String> expected = new LinkedHashSet<>(table.getDefinition().getColumnNameSet());
-            expected.removeAll(usedNames);
-            throw new IllegalStateException("Not all columns were populated, missing " + expected);
-        }
+        NoSuchColumnException.throwIf(
+                table.getDefinition().getColumnNameSet(),
```

Comment:
I think this is backwards, and I think the description requires revision. This complains if we didn't use all of the table columns, so you want available to be `usedColumns` and requested to be the definition's columns.

Reply:
I think you are correct; I had this backwards (tests still pass b/c they actually need to be equal). That said, I think the error message is still correct.